txauthor.go

```diff
@@ -184,3 +186,136 @@ func (lw *LibWallet) PublishUnminedTransactions() error {
 	err = lw.wallet.PublishUnminedTransactions(ctx, netBackend)
 	return err
 }
+
+func (lw *LibWallet) SendFromCustomInputs(sourceAccount uint32, requiredConfirmations int32, utxoKeys []string,
+	txDestinations []txhelper.TransactionDestination, changeDestinations []txhelper.TransactionDestination, privPass []byte) (string, error) {
+
+	// fetch all utxos in account to extract details for the utxos selected by user
+	// use targetAmount = 0 to fetch ALL utxos in account
```

Comment:
This comment is no longer relevant here. Should be moved to before line 86 of `accounts.go`. Also, rename the `UnspentOutputs` function to `AllUnspentOutputs`